documentation/modules/ROOT/pages/03-demo.adoc

```diff
@@ -7,28 +7,879 @@
 
 == Demonstration
 
-Include here content related to potential existing demos: blogs, articles, recorded videos, walkthrough guides, tutorials.
+Let's take a look at the following example.
+We'll be looking at a Cabs ride-sharing application, that mimics real-world solutions of similar kind.
```

Comment:
I;d not say that we want to mimic something from the real-world.

Reply:
Do you propose dropping the second sentence? Or re-phrase it?

Reply:
just a different word, instead of mimic?